public/index.html

```diff
@@ -2,42 +2,25 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="author" content="Manoj Asarsa" />
+    <meta name="keywords" content="easyBuy, easybuyy e-commerce" />
     <meta name="theme-color" content="#000000" />
```

Comment:
You can actually give your theme color here. It will look good.

Reply:
That's a good idea. I'll do that.